image/rhel/konflux.Dockerfile

```diff
@@ -1,7 +1,28 @@
+FROM registry.access.redhat.com/ubi8/nodejs-18:latest AS ui-builder
+
+# Switch to root because ubi8/nodejs image runs as non-root by default which does not let e.g. install yarn.
+USER 0:0
+
+WORKDIR /go/src/github.com/stackrox/rox/app
+
+COPY . .
+
+# Install yarn
+RUN make -C ui/.yarn-dist install
+
+# This sets branding during UI build time. This is to make sure UI is branded as commercial RHACS (not StackRox).
+# ROX_PRODUCT_BRANDING is also set in the resulting image so that Central Go code knows its RHACS.
+# TODO(ROX-22338): switch branding to RHACS_BRANDING when intermediate Konflux repos aren't public.
```

Comment:
> intermediate Konflux repos

by that, do you mean the `quay.io/user-workload/...` repos?

Reply:
Probably yes, I meant where built containers are immediately published: `quay.io/redhat-user-workloads/rh-acs-tenant/acs/main`. These are open for public reading, right?

Reply:
> These are open for public reading, right?

Yes, they are.